src/go/k8s/hack/install-cert-manager.sh

```diff
@@ -24,7 +24,7 @@ else
 fi
 
 HOME=$(mktemp -d)
-helm repo add jetstack https://charts.jetstack.io
-helm repo update
-helm install cert-manager jetstack/cert-manager --version "${CERT_MANAGER_VERSION}" --set installCRDs=true --namespace cert-manager --create-namespace
-./bin/cm-verifier --timeout 5m
+# the recommended approach using a static manifest:
```

Comment:
NIT: From the https://cert-manager.io/docs/installation/ page I don't see flux maintainers recommend using static manifest.